plugins/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/requirement/OpenShiftCommandLineToolsRequirement.java

```diff
@@ -56,12 +61,21 @@ public boolean canFulfill() {
 
 	@Override
 	public void fulfill() {
-		File ocBinary = new File("binaries" + SEPARATOR + "oc");
-		if (!ocBinary.exists()) {
-			downloadAndExtractOpenShiftClient();
+		if (!OCBinaryFile.get().getFile().exists()) {
+			File downloadedOCBinary = downloadAndExtractOpenShiftClient();
+			createSymlink(downloadedOCBinary);
 		} else {
 			logger.info("Binary is already downloaded.");
-			PATH_TO_CLIENT = ocBinary.getAbsolutePath(); 
+		}
+	}
+
+	private void createSymlink(File downloadedOCBinary) {
+		try {
+			Files.deleteIfExists(OCBinaryFile.get().getFile().toPath());
```

Comment:
this will never succeed, because createSymlink run only if oc binary file does not exist and thus removing such file is pointless, if it does not exist.